### PR TITLE
Add examples section to readme

### PR DESCRIPTION
**Pull request by ChrisixFlash**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,10 +189,71 @@ For more documentation about PSPDFKit check out [PSPDFKit online documentation](
...
+### Xamarin.Forms
+
+`XFSample.Droid` is an example showcasing how to build an app with PSPDFKit using Xamarin.Forms.
+
+<img width="80%" src="Images/XForms.png"/>
+
+
+### AndroidSimple
+
+`AndroidSimple` is an example project showing showing how to open a preset document, or how to open a document from various file providers or the local file directory.
```
Comment (radazzouz) [inline README.md]: "to open a preset document" -> "to open **and** preset document"

**Review by radazzouz — APPROVED** [review]
Minor nitpick. Otherwise, it looks great! 👍

File: README.md (hunk 1 above)
Comment (ChrisixFlash) [reply README.md]: `Preset` is meant as a document that is set/declared previously. I can see how that word choice could confuse customers though 😕 
We could write your version though and go with "to open and present a document", which one would you prefer?

Comment (radazzouz) [reply README.md]: I misread "preset" for "present". 🤦‍♂️ 

In this context, I'd go with "to open and present a document"

**Pull request merged**